set.mm

```diff
@@ -476170,6 +476181,105 @@ The sumset operation can be used for both group (additive) operations and
       TUUHYTUUIUUFUAUBUUPCAUUTUVIUXGUUGR $.
   $}
 
+  ${
+    ssmxidl.1 $e |- B = ( Base ` R ) $.
+    ${
+      $d B a b i x $.  $d B j p $.  $d I j p $.  $d R a b i x $.  $d R j p $.
+      $d Z a b i x $.  $d Z j p $.  $d a b i j ph x $.
+      ssmxidllem.1 $e |- P = { p e. ( LIdeal ` R ) | ( p =/= B /\ I C_ p ) } $.
+      ssmxidllem.2 $e |- ( ph -> R e. Ring ) $.
+      ssmxidllem.3 $e |- ( ph -> I e. ( LIdeal ` R ) ) $.
+      ssmxidllem.4 $e |- ( ph -> I =/= B ) $.
+      ssmxidllem2.1 $e |- ( ph -> Z C_ P ) $.
+      ssmxidllem2.2 $e |- ( ph -> Z =/= (/) ) $.
+      ssmxidllem2.3 $e |- ( ph -> [C.] Or Z ) $.
+      $( The set used in the proof of ~ ssmxidl satisfies the condition of
```

Comment:
Which set is this? `P `? Then the comment should start with "The set ` P ` used in the proof... ".

Reply:
You guessed it, it's `P`! 
Added.